webapp/src/main/java/com/box/l10n/mojito/utils/StatsDMetricsConfig.java

```diff
@@ -0,0 +1,94 @@
+package com.box.l10n.mojito.utils;
+
+import org.springframework.boot.actuate.autoconfigure.metrics.MeterRegistryCustomizer;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.lang.Nullable;
+
+/**
+ * StatsD metrics related config
+ */
+@Component
+@ConfigurationProperties("l10n.management.metrics.export.statsd")
+@ConditionalOnProperty(value="management.metrics.export.statsd.enabled", havingValue="true")
+public class StatsDMetricsConfig {
+
+    private NamingTransformation nameTransformation = NamingTransformation.NONE;
+
+    private String tagKeyPrefix = "";
+
+    private String tagValuePrefix = "";
+
+    private String namePrefix = "";
+
+    public NamingTransformation getNameTransformation() {
```

Comment:
do we really need to support name transformation? can't we just do adding prefix on top of an existing flavors?